proxy.py

```diff
@@ -28,8 +29,11 @@ async def netcat(port):
     # Verbose mode
     if os.environ["VERBOSE"] == "1":
         command.append("-v")
-    command += [f"{mode}-listen:{port},fork,reuseaddr,max-children={max_connections}",
-                f"{mode}-connect:{ip}:{port}"]
+    if mode.lower() == "udp" and udp_answers == "0":
```

Comment:
Oops sorry I thought you removed this.

```suggestion
    if mode == "udp" and udp_answers == "0":
```

I know it seems silly, but to be correct, testiing that `.lower()` works would mean setting up an array of tests with udp, UDP, Udp, UdP, uDp, etc. I just prefer to have one single valid value and fail otherwise.

Since it is documented, it shouldn't be a problem downstream.

Reply:
it will not fail otherwise, it's just silently ignored.